pkg/debug/checkups/root_directory.go

```diff
@@ -0,0 +1,58 @@
+package checkups
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/kolide/launcher/pkg/agent/types"
+)
+
+type RootDirectory struct {
+	k       types.Knapsack
+	status  Status
+	summary string
+}
+
+func (c *RootDirectory) Name() string {
+	return "Root directory contents"
+}
+
+func (c *RootDirectory) Run(_ context.Context, fullFH io.Writer) error {
+	files, err := os.ReadDir(c.k.RootDirectory())
+	switch {
+	case errors.Is(err, os.ErrNotExist):
+		c.status = Failing
+		c.summary = fmt.Sprintf("root directory (%s) not present", c.k.RootDirectory())
+	case err != nil:
+		c.status = Erroring
+		c.summary = fmt.Sprintf("listing files in root directory (%s): %s", c.k.RootDirectory(), err)
+	case len(files) == 0:
+		c.status = Warning
+		c.summary = fmt.Sprintf("root directory (%s) empty", c.k.RootDirectory())
+	default:
+		c.status = Passing
+		c.summary = fmt.Sprintf("root directory (%s) contains %d files", c.k.RootDirectory(), len(files))
+
+		fmt.Fprintf(fullFH, "%s\n", files)
+	}
+	return nil
+}
+
+func (c *RootDirectory) ExtraFileName() string {
+	return "file-list"
```

Comment:
Other `ExtraFileName`s have an extension -- should this one?

Reply:
Kinda up to us. They're named whatever we think is easy to understand.

Mostly, I started with calling them `full` and decided that was confusing since processes bubbled up a list, while logs wanted to make a zip file. I could decide on an extension here, so I skipped it. posix doesn't require one